app/presenters/schools/users/show_presenter.rb

```diff
@@ -0,0 +1,112 @@
+module Schools
+  module Users
+    class ShowPresenter < ApplicationPresenter
+      attr_reader :user
+
+      def initialize(view_context, user)
+        @user = user
+        super(view_context)
+      end
+
+      def avatar_url
+        user.avatar_url || user.initials_avatar
+      end
+
+      def courses_taken
+        @courses_taken ||=
+          begin
+            Course
+              .joins(cohorts: { students: :user })
+              .where(users: { id: user.id })
+              .order(name: :asc)
+              .distinct
+          end
```

Comment:
A lot of these methods seem to have unnecessary `begin-end` wrapping. We usually use that when we have a long multi-line expression to write, whose return needs to be assigned to a variable (or similar).